src/server/live-reload/watcher.js

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+
+var config = require('../config');
+var fs = require('fs');
+var log = require('../log');
+var path = require('path');
+var Q = require('q');
+
+var eventIgnoreDuration = 150;
+var wwwRoot = 'www';
+var mergesRoot = 'merges';
+var unknownFileId = '__sim-unknown__';
```

Comment:
If these variables are constants, then we should have some convention about it. Maybe using uppercase letters for the names.